internal/deb/extract.go

```diff
@@ -38,6 +38,13 @@ type ExtractInfo struct {
 	Context  any
 }
 
+type HardLinkInfo struct {
+	// The path of the hard link recorded in the tarball header.
+	LinkPath string
```

Comment:
What naming convention? I cannot see anything in the man page that refers to "link path".

Reply:
It's `LINK_NAME`. Since this variable here represents a path, using `LinkPath` would be more clear than `TargetPath`, as this would easily got mixed with `TARGET` for `ln`.